### ISSUE TITLE
Not able to change the frameColor

### ISSUE THREAD

**Issue by YahavTheKing123** [spatialillusions/milsymbol#238]
Hi Mans,

I am trying to change the default black frame around the icon but it doesnt work for me.
I am inserting the options obj the following:

var sym = new ms.Symbol('someSidc');
sym.setOptions({frameColor: '#b37eff'});
sym.asSVG();

But I see that the frameColor is not being changed :(

What do I missing here?

Thanks for your reply!

**Comment by YahavTheKing123** [comment]
I found the solution. I inserted the color this way:
options.frameColor= {
Civilian: "red",
Friend: "red",
Hostile: "red",
Neutral: "red",
Unknown: "red",
}

and it works. 

Still I have one question:
for the other modifiers, i can see that they are also being colored in the same color.
Can i preseved the black color for the modifier while I am changging only the frame color?

Thanks in advance!

![image](https://user-images.githubusercontent.com/38849753/68862858-e0b68180-06f6-11ea-8cc0-45e140e6aa2f.png)
